frontend/src/utils/util.js

```diff
@@ -1,11 +1,11 @@
 export const convertPaginationToParams = (pagination) => {
-  let result = "";
+  let result = '';
   if (
```

Comment:
I think, you could rewrite this logic `if (!pagination || (!pagination.key && !pagination.limit)) {
  return '';
}`